Update the contributing guidelines (#1174)
LawnchairLauncher/lawnicons: .github/CONTRIBUTING.md

Each icon must fit the 160x160px or 144x144px (depending on the shape) content area size. It must not be smaller nor bigger than the specified sizes.

> The default stroke should be kept at 12px. For fine detail a stroke of 6px can be used. The standard widths are 6px, 8px, 10px, 12px and 14px.

Reviewer: ```suggestion
The default stroke should be kept at 12px. For fine detail, a stroke of 6px can be used. The standard widths are 6px, 8px, 10px, 12px and 14px.
```

> Note that the 14px width should be used if an icon is too minimal or does not have enough surrounding details. You can check whether it should use 14px if the stroke width looks too thin alongside other icons. (More information in the [Figma document](https://www.figma.com/community/file/1227718471680779613))

Reviewer: ```suggestion
If an icon is too minimal or does not have enough surrounding details, 14px should be used. You can check whether to use it if the stroke width looks too thin alongside other icons. ([More information in the Figma document](https://www.figma.com/community/file/1227718471680779613))
```

> In addition to the above, the icons must have an outlined (not filled) style and the colour must be black (**#FFFFFF**). If the original icon has a filled style, you should change the icon to adhere to the guidelines as seen below.

Reviewer: Black is `#000000`, white is `#ffffff`. Also I'd recommend using `codeblocks`:

```suggestion
In addition to the above, the icons must have an outlined (not filled) style and the colour must be black `#000000`. If the original icon has a filled style, change the icon to adhere to the guidelines as seen below.
```

Author: Completely mb there. Thanks
